src/Client.cpp

```diff
@@ -73,7 +73,10 @@ CClient::~CClient() {
 		CClientAuth* pAuth = (CClientAuth*) &(*m_spAuth);
 		pAuth->Invalidate();
 	}
-	if (m_pUser != nullptr) {
+	if (m_pNetwork != nullptr) {
```

Comment:
btw, i'm not so sure if client side traffic should be associated to any network or not